Backend/starter_project_dotnetforce7/BlogApp2/BlogApp.Application/Features/Blog/CQRS/Commands/CreateBlogCommand.cs

```diff
@@ -0,0 +1,16 @@
+using BlogApp.Application.Features.Blogs.DTOs;
+using BlogApp.Application.Responses;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Features.Blogs.CQRS.Commands
+{
+    public class CreateBlogCommand: IRequest<BaseCommandResponse>
```

Comment:
It's okay to return baseCommandResponse instance depending your on what you need to return but I have seen that your group have implemented response type called "Result". If you have communicated within group and decided to use baseCommandResponse here that's fine...but for consistence you could also change this to return "Result" instance.

The same applies for other CQRS commands and queries.